Regression/WarpX-tests.ini

```diff
@@ -2044,7 +2044,7 @@ dim = 3
 addToCompileString = USE_PSATD=TRUE
 restartTest = 0
 useMPI = 1
-numprocs = 1
+numprocs = 2
```

Comment:
If you optimize for speed here, increasing `numthreads` instead of `numprocs` might be faster (subject to testing ^^)

Reply:
I tried locally and I recorded longer runtimes with 1 MPI process and 2 openMP threads, rather than 2 MPI processes and 1 openMP thread. So I think I'll keep the changes as they are. I will reset the benchmarks and ping Andrew on the `particle_cpu` issue.